testing/prow/gpu-operator.sh

```diff
@@ -81,11 +81,21 @@ prepare_cluster_for_gpu_operator() {
 
     ${THIS_DIR}/entitle.sh
 
+    if [ "${USE_PREV_RELEASE_CATALOG:-}" ]; then
+        # Deploy OCP version N-1 catalog - Usually needed for testing on OCP next.
+        _warning "NFD_Operator_deployed_from_previous" "The NFD Operator was deployed from the version N-1 of the catalog (USE_PREV_RELEASE_CATALOG variable was set)"
+        ./run_toolbox.py cluster deploy_previous_catalog_source redhat-operators
+        OPERATOR_CATALOG="--catalog redhat-operators-previous"
```

Comment:
I think you can change this `"--catalog redhat-operators-previous"` to `"--catalog=redhat-operators-previous"` and remove the `$(echo $OPERATOR_CATALOG | xargs)` -> `$OPERATOR_CATALOG`

Reply:
Right 👏🏽